wire-ios-data-model/Source/ManagedObjectContext/CoreDataStack.swift

```diff
@@ -233,6 +234,9 @@ public class CoreDataStack: NSObject, ContextProvider {
                     }
                     return
                 }
+                if tp?.warnIfLongerThanInterval() != true {
```

Comment:
I've added one commit to remove optionality in 785082f05f4aadedd5e1d1f6813274b62ee624cf, because I didn't understand the result of the `optional != true`.